### PR TITLE
Fix deprecated set-output in Github Actions

### PR DESCRIPTION
**Pull request by McGiverGim**
Fixes https://github.com/ExpressLRS/ExpressLRS-Configurator/issues/446

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
 
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```
Comment (jurgelenas) [inline .github/workflows/publish.yml:28]: I think you are not retrieving this in 

```
      - name: Get yarn cache directory path
        id: yarn-cache-dir-path
        run: echo "::set-output name=dir::$(yarn cache dir)"

      - uses: actions/cache@v3
        id: yarn-cache
        with:
          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
          restore-keys: |
            ${{ runner.os }}-yarn-
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
```
Comment (jurgelenas) [inline .github/workflows/test.yml:37]: Need to retrieve this in `steps.yarn-cache-dir-path.outputs.dir`.

**Review by jurgelenas — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (McGiverGim) [reply .github/workflows/publish.yml:28]: Hi! I'm not too sure if I understand the problem.

This is the execution before the change:
image


This is the execution after the change:
image

It seems the output is being retrieved without problem.

Comment (jurgelenas) [reply .github/workflows/publish.yml:28]: Yes, it is actually good. Thanks.

**Review by jurgelenas — APPROVED** [review]

**Pull request merged**